src/form/field/message/FormFieldMessage.tsx

```diff
@@ -1,27 +1,27 @@
-import "./_form-field-message.scss";
-
-import React from "react";
-import classNames from "classnames";
-
-interface FormFieldMessageProps {
-  className?: string;
-  type: "error" | "warning" | "helper";
-  message?: string;
-  testid?: string;
-}
-
-function FormFieldMessage({className, type, message, testid}: FormFieldMessageProps) {
-  const messageRowClassName = classNames(
-    "form-field-message",
-    className,
-    `form-field-message--is-${type}`
-  );
-
-  return (
-    <p data-testid={testid} className={messageRowClassName}>
-      {message}
-    </p>
-  );
-}
-
-export default FormFieldMessage;
+import "./_form-field-message.scss";
+
+import React from "react";
+import classNames from "classnames";
+
+export interface FormFieldMessageProps {
+  className?: string;
+  type: "error" | "warning" | "helper";
+  message?: string;
+  testid?: string;
+}
+
+function FormFieldMessage({className, type, message, testid}: FormFieldMessageProps) {
+  const messageRowClassName = classNames(
+    "form-field-message",
+    className,
+    `form-field-message--is-${type}`
+  );
+
+  return (
+    <p data-testid={testid} className={messageRowClassName}>

```

Comment:
No need to define this variable
```suggestion
  return (
    <p data-testid={testid} className={classNames(
    "form-field-message",
    className,
    `form-field-message--is-${type}`
  )}>

```